plugin.php

```diff
@@ -295,6 +295,18 @@ function rest_validate_request_arg( $value, $request, $param ) {
 			}
 		}
 
+		if ( 'array' === $args['type'] ) {
+			// A comma-separated array of IDs could be acceptable instead of an array.
```

Comment:
> JSON Schema permits us to specify two different types for a given property. In the case of anything using `wp_parse_id_list()`, we need to specify `type => [ 'string', 'array' ]`, and then properly validate an array of types.

I just want to make sure I'm understanding you correctly. In the case of `type => [ 'string', 'array' ]`, we should ideally be checking for an array, and then ensuring that each element is a string? Or as another example, `type => [ 'integer', 'array' ]` would mean checking for an array of integers?